scripts/release/setup.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+#
+# Externally settable env vars:
+# GIT_REPO_PATH= something to `git clone` from
+# GIT_CHECKOUT_LABEL= something to `git checkout` and build from (branch or tag or hash)
+
+if [ -z "${BUILDTIMESTAMP}" ]; then
+    date "+%Y%m%d_%H%M%S" > "${HOME}/buildtimestamp"
+    BUILDTIMESTAMP=$(cat "${HOME}/buildtimestamp")
+    export BUILDTIMESTAMP
+    echo run "${0}" with output to "${HOME}/buildlog_${BUILDTIMESTAMP}"
+    (bash "${0}" 2>&1) | tee "${HOME}/buildlog_${BUILDTIMESTAMP}"
+    exit 0
+fi
+
+date "+setup start %Y%m%d_%H%M%S"
+
+set -ex
+
+if [ -z "${GIT_REPO_PATH}" ]; then
+    GIT_REPO_PATH=https://github.com/btoll/go-algorand
+fi
+
+if [ -z "${GIT_CHECKOUT_LABEL}" ]; then
+    GIT_CHECKOUT_LABEL="rel/stable"
+fi
+
+export DEBIAN_FRONTEND=noninteractive
+
+sudo apt-get update -q
+sudo apt-get upgrade -q -y
+
+# Some of these dirs aren't used until later scripts.
+#umask 0077
+#mkdir -p ~/.gnupg
+#mkdir -p "${HOME}"/{.gnupg,go,gpgbin,docker_test_resources,dummyaptly,dummyrepo,prodrepo}
+mkdir -p "${HOME}"/{.gnupg,go,gpgbin,dummyaptly,dummyrepo,prodrepo}
+
+export GOPATH=${HOME}/go
```

Comment:
is there any particular reason you're not using 
```
GOPATH=$(go env GOPATH)
export GOPATH
```
?

Reply:
I remember at one point that it couldn't find the go binary and so I manually set it to keep moving.  I'll revisit this.